### ISSUE TITLE
drag and drop pdf/url dosent seem to work on linux

### ISSUE THREAD

**Issue by zeltak** [jkitchin/org-ref#92]
Hi John

I noticed after watching your excellent video that Drag and drop a pdf as well as a url dosent seem to work. this is the message i get after dragging a url
```
Error during redisplay: (#[(window object position) "\212b\210\303 \304\305!r
q\210\306\216	c\210\307 \210\310 -\207" [position s temp-buffer org-ref-get-citation-string-at-point generate-new-buffer " *temp*" ((byte-code "\301!\203
\302!\210\301\207" [temp-buffer buffer-name kill-buffer] 2)) fill-paragraph buffer-string] 2] #<window 5 on citetst.org> #<buffer citetst.org> 215) signaled (wrong-type-argument stringp nil)
Mark set
5 1
```

and for the pdf dragging it into a bib buffer just opens the pdf into doc-view

thx again

Z

**Comment by jkitchin** [comment]
Those errors don't make much sense to me. I can't tell if you dragged a pdf onto an org file maybe? It should only work on a bibtex file. Your error looks like the code that is making a tooltip somehow.

The second issue suggests drag-n-drop isn't configured for the org-ref-pdf code. Did you (require 'org-ref-pdf)? This is not loaded by default.

I won't have access to a Linux machine for testing until early Jan. and I  can take a look at it then.

**Comment by TimoLassmann** [comment]
Hi John, 

I've been having similar issues. After requiring org-ref-pdf and pointing emacs to pdftotext (I'm on a mac) I am getting:
No doi found in file:/Users/tlassmann/Downloads/acscatalE5b00538.pdf
I manually converted this file with pdftotext and the doi is definitely there... 
Any ideas? 
thanks!

**Comment by jkitchin** [comment]
@zeltak I tested this on Linux, and the drag and drop works exactly the way I would expect it to. I will need more information to help you figure it out.

**Comment by jkitchin** [comment]
@TimoLassmann I pushed a debug function here https://github.com/jkitchin/org-ref/blob/master/org-ref-pdf.el#L201 that might help you figure out why no  DOI is being found. Could you try it out?

**Comment by TimoLassmann** [comment]
Hi John, 

As an emacs newbie I simply did not load all the packages distributed in org-ref in my emacs config. 

By comparing my installation to your jmax setup I was able to get everything working. For example I noticed I need to load “cl” package to get the crossref-add-bibtex-entry function to work properly. 

All around great package and exactly what I have been looking for in a long while. 

To be really useful in life sciences I will try to add org-ref-url-scrapers for journals like Genome Research, Bioinformatics etc.. 

Cheers,

Timo
